### DISCUSSION TITLE
Drain from a cauldron/dripstone

### DISCUSSION THREAD

**Discussion by Aton-Kish** [TechReborn/TechReborn#2536] [Ideas]
**Is your feature request related to a problem? Please describe.**
The drain block cannot pull from a cauldron and fill from a dripstone directly.

**Describe the solution you'd like**
Drain from a cauldron/dripstone.

**Describe alternatives you've considered**
Perhaps it would work well to add a cauldron and dripstone conditions to the below codes.
<https://github.com/TechReborn/TechReborn/blob/12765ae87fca6c853b157f534b395ebac80977ec/src/main/java/techreborn/blockentity/machine/misc/DrainBlockEntity.java#L85-L93>

**Additional context**
We will be able to build the automatic dripstone fluid farm.
![lava_farm](https://user-images.githubusercontent.com/38515249/124375001-907ca080-dcda-11eb-8117-19e704c4472e.png)

**Comment by Ayutac** [comment]
I would only take from Cauldron, as the other part would need to reimplement the dripstone filling logic I guess

**Comment by Technici4n** [comment]
Will be addressed by the Fabric Transfer API.
